### ISSUE TITLE
Feature request: User-Agent and Referer in HTTP query

### ISSUE THREAD

**Issue by JuttaAnders** [tumic0/GPXSee#196]
Would it be possible to add an entry in the **map definition** to set a user defined "User-Agent" and "Referer" for HTTP requests?

I would like to deal with a French WMS server (www.ign.fr) which supports user identification by sending a certain "User-Agent" and/or "Referer" with every HTTP request.

**Comment by tumic0** [comment]
Well, technically this should not be a big issue, but I'm really not sure if I want to add such feature. For authorization the HTTP authorization header should be used (or a key in the URL). The user-agent has a different purpose, not even to mention the referer header.

What exactly are those IGN services? The IGN services in the GPXSee-maps repository work with URL keys as far as I remember.

**Comment by JuttaAnders** [comment]
To access IGN I had to register an account which I shared with two other persons.
The account has a certain "free" limit (quota) before I have to start to pay, or the server will stop sending images until the next month (or something like that).

The key in the URL would the same for all 3 of us, but in the settings of the account at IGN it is possible to define other kinds of identification (user-agent and/or referrer) that is unique (and must be kept secret by the user of course) for each user.

So each user has its own "quota", without influencing the others...
